.github/workflows/release-code-freeze.yml

```diff
@@ -153,19 +153,39 @@ jobs:
       branch: trunk
       bump-type: dev
 
-  build-zip:
-    name: Build WooCommerce release ZIP
+  build-dev-release:
+    name: 'Build WooCommerce -dev release'
     uses: ./.github/workflows/release-build-zip-file.yml
     needs: [prepare-for-code-freeze, run-code-freeze]
     with:
       skip_verify: true
-      create_github_release: false
+      create_github_release: true
       branch: ${{ needs.prepare-for-code-freeze.outputs.nextReleaseBranch }}
 
+  publish-dev-release:
+    name: 'Publish -dev release'
+    runs-on: ${{ ( github.repository == 'woocommerce/woocommerce' && 'blacksmith-2vcpu-ubuntu-2404' ) || 'ubuntu-latest' }}
+    needs: [prepare-for-code-freeze, build-dev-release]
+    outputs:
+      release-zip: ${{ steps.publish-release.outputs.release-zip }}
+    steps:
+      - id: publish-release
+        run: |
+          gh release edit '${{ needs.prepare-for-code-freeze.outputs.nextReleaseVersion }}-dev' \
+            --prerelease \
+            --latest=false \
+            --notes='' \
+            --draft=false
+
+          echo "release-zip=https://github.com/${{ github.repository }}/releases/download/${{ needs.prepare-for-code-freeze.outputs.nextReleaseVersion }}-dev/woocommerce.zip" >> $GITHUB_OUTPUT
+        env:
```

Comment:
Nit: it's a bit easier to read when `env` is placed before `run` -  somehow my brain respects that the bash code to execute is the latest thing in step definition.

Reply:
Fixed in 62ddf0af1ab8858a365db383b891978bc7a240bd.